Sources/BraintreeCard/Public/BraintreeCard/BTCardClient.h

```diff
@@ -26,6 +26,9 @@ typedef NS_ENUM(NSInteger, BTCardClientErrorType) {
 
     /// Customer provided invalid input
     BTCardClientErrorTypeCustomerInputInvalid,
+    
+    /// Card already exists as a saved payment method
+    BTCardClientErrorTypeCardAlreadyExists,
```

Comment:
Pending discussion of whether or not this should be considered a breaking change

Reply:
I don't think this is a breaking change. As long as the merchant is handling a catch all error of "something went wrong" or just passing back the error message from the error directly, it should be fine.